src/client/containers/ResetPassword/index.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react';
+import { resetPassword } from '../../firebase/auth';
+import ResetPassword from '../../components/Forms/ResetPassword';
+import Loader from '../../components/Loader';
+
+export default function SignInContainer() {
+  const [isLoading, setIsLoading] = useState(false);
+
+  const onSubmit = async ({ email }) => {
```

Comment:
For the question above i guess its to not do the prevent defaults here, which makes sense

Reply:
yes :) is unrelated to the logic but to UI :) so agree :)